pkg/apis/v1/labels.go

```diff
@@ -53,6 +53,7 @@ const (
 	NodePoolHashAnnotationKey                  = apis.Group + "/nodepool-hash"
 	NodePoolHashVersionAnnotationKey           = apis.Group + "/nodepool-hash-version"
 	NodeClaimTerminationTimestampAnnotationKey = apis.Group + "/nodeclaim-termination-timestamp"
+	NodeClaimMinValuesAutoRelaxedAnnotationKey = apis.Group + "/nodeclaim-min-values-auto-relaxed"
```

Comment:
I'm curious why we need this annotation? You can't query on annotations so I'm curious how we expect this being added to the NodeClaim to be helpful